bundles/core/src/main/java/com/adobe/cq/commerce/core/components/internal/models/v1/categorylist/FeaturedCategoryListImpl.java

```diff
@@ -76,6 +77,7 @@ public class FeaturedCategoryListImpl extends DataLayerComponent implements Feat
     private static final String CATEGORY_IDENTIFIER = "categoryId";
     private static final String ASSET_PROP = "asset";
     private static final String ITEMS_PROP = "items";
+    private static final String CATEGORY_IDENTIFIER_TYPE = "categoryIdType";
```

Comment:
You may want to put this on the public interface like here: https://github.com/adobe/aem-core-cif-components/pull/1011/files#diff-5e1c4c18538024ef6a86682d30b765717aeea3d4af9372f76de7b34190c67afdR44